### PR TITLE
Navigate to inventory list after create

### PR DESCRIPTION
**Pull request by vatthikorn**

### CODE REVIEW

File: src/Inventory/Home.js [hunk 1]
```diff
@@ -35,14 +37,49 @@ export default class Home extends Component {
...
+						onPress={params.showNewListInput}
 						containerStyle={styles.rightButton}
 					/>
 				</View>
 			)
 		}
 	}
 
+	componentWillMount() {
+		this.props.navigation.setParams({ showNewListInput: this.showNewListInput });
```
Comment (indirap) [inline src/Inventory/Home.js:49]: What is this for?

File: src/Inventory/Home.js [hunk 2]
```diff
@@ -35,14 +37,49 @@ export default class Home extends Component {
...
+	}
+
+	showNewListInput() {
+		this.setState(prevState => ({ newListInputVisible: !prevState.newListInputVisible }));
```
Comment (indirap) [inline src/Inventory/Home.js]: Can we put this on multiple lines for readability?

(And perhaps I do this too... if so I'll update the stuff I wrote 🙂)

File: src/Inventory/Home.js [hunk 3]
```diff
@@ -35,14 +37,49 @@ export default class Home extends Component {
...
+	}
+
+	addNewList() {
+
+		if (this.state.newListName.length === 0) {
+			console.log('name cannot be empty') // eslint-disable-line
```
Comment (indirap) [inline src/Inventory/Home.js]: Remove this?

File: src/Inventory/Home.js (hunk 1 above)
Comment (vatthikorn) [reply src/Inventory/Home.js:49]: It's to allow the header button to work with its screen components. I got it from the documentation [here](https://reactnavigation.org/docs/en/header-buttons.html#header-interaction-with-its-screen-component)

**Review by indirap — APPROVED** [review]

**Pull request merged**